warehouse/warehouse.go

```diff
@@ -187,7 +187,7 @@ func (wh *HandleT) backendConfigSubscriber() {
 }
 
 func (wh *HandleT) syncLiveWarehouseStatus(sourceID string, destinationID string) {
-	rows, err := wh.dbHandle.Query(fmt.Sprintf(`select id from %s where source_id='%s' and destination_id='%s' order by updated_at asc limit %d`, warehouseutils.WarehouseUploadsTable, sourceID, destinationID, warehouseSyncPreFetchCount))
+	rows, err := wh.dbHandle.Query(fmt.Sprintf(`select id from (select id from %s where source_id='%s' and destination_id='%s' order by updated_at desc limit %d) as _ order by updated_at asc`, warehouseutils.WarehouseUploadsTable, sourceID, destinationID, warehouseSyncPreFetchCount))
 	if err != nil && err != sql.ErrNoRows {
```

Comment:
can we iterate the returned results in array instead of doing it in sql?

Reply:
okay